keyboards/ducky/one2sf/1967st/config.h

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2019 /u/KeepItUnder
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x0416
+#define PRODUCT_ID      0x0123
```

Comment:
https://www.the-sz.com/products/usbid/index.php?v=0x0416&p=0x0123&n=

This vendor appears to belong to nuvaton, but not actually an assigned product ID. 

I don't feel comfortable about using it, in this case, and would prefer an unused vendor ID.

Reply:
so, i just put random vendor_id that unused by any vendor? or there are any regulation for that?